otel-extension/src/main/java/akka/actor/impl/LocalActorRefProviderAdvice.java

```diff
@@ -0,0 +1,12 @@
+package akka.actor.impl;
+
+import io.scalac.mesmer.otelextension.instrumentations.akka.actor.InstrumentsProvider;
+import net.bytebuddy.asm.Advice;
+
+public class LocalActorRefProviderAdvice {
+
+  @Advice.OnMethodExit
+  public static void actorOfExit() {
+    InstrumentsProvider.instance().actorsCreated().add(1);
```

Comment:
The advice is something that will attach our mesmer code to akka LocalActorRefProvider class (a bytebuddy concept called "advice"). The instrument provider is just something that will provide an actorsCreated counter so that we could increase it.